source/chapters/appendix/mixxx_controls.rst

```diff
@@ -3183,7 +3183,14 @@ Note that :mixxx:coref:`[Library],MoveUp` and other Move and Scroll controls emu
 
 .. mixxx:control:: [Library],GoToItem
 
-   Equivalent to double clicking the currently selected item
+   Triggers different actions, deepending on which interface element currently has keyboard focus:
+
+   | **Search bar** text box: moves focus to tracks table
```

Comment:
The line break preservation is hacky. This should either be a definition list or an enumeration (unordered list) IMHO.

Another alternative would be a two-column table.

Reply:
hmm neither really fits IMO.
I'll make it a table
![image](https://user-images.githubusercontent.com/5934199/137407162-a8bc9649-f8a7-4e7c-8a21-ad77f4785735.png)